### ISSUE TITLE
Pause reviews

### ISSUE THREAD

**Issue by JuTrip** [fabd/kanji-koohii#160]
Hello,

I hope I am not asking a feature related question already answered in the past in the forum (I browsed the topics here and saw nothing)

Would it be possible to add a "stop" feature to pause the daily addition of new cards to review? 

Like everybody else I am sure, I sometimes can not review kanjis for a few days or weeks and it feels unmotivating to have a stack of hundreds of cards waiting for me when I resume my studies.

Thanks for your answer

**Comment by talmobi** [comment]
The number of Kanjis decrease over time. I don't think this is a good idea. If you have a buildup of 100 or so Kanjis, you can only tackle like 10 at a time each day. But you will catch up because the number of Kanjis added will decrease over time as you start learning them.

**Comment by JuTrip** [comment]
Hello !

My problem is not about having the kanjis "learned" and their amount "decreasing over time", but rather the amount of them coming back at the same time. I should give a bit of context to my request so it's better understood.

I have been using this website since 2011 and have learned the 2122 characters of the 1st book in about 3 months at that time. Since then, I have been reviewing more or less frequently, trying to keep the "review"  stack under 30 cards (and the restudy around 50). Today, (with 7+ years of learning) 80% of my review cards are coming back in 7+ weeks (as shown by the last bar of the barplot). 

My problem
Although they come back for review much more spaced than at the beginning, every 6 months or so now, they come in waves of 15+ per day for about 3 months, corresponding to the period when I learned them. 

As I have less time nowadays than when I was studying in 2011, I often skip days of review now and end up with hundreds of them to review, whence my frustration.

I hope I could make myself clearer this time (sorry but English is not my mother tongue).

Thanks

**Comment by fabd** [comment]
I want to solve this eventually.

I think what's missing is a form of "throttling" of due cards, which can be greater for cards with longer interval, and smaller or even off at lower intervals. Hence if learning 100 new cards, you'd want to review all of them in the following days (they're new) -- but the same batch of cards months later could be throttled to a configurable "due cards per day" setting.

I'm not sure how the setting would work. Some kind of setting would scale this "throttling" of due cards from low to high intervals, and maybe a checkbox option to enforce a fixed due cards per day value -- no matter if the cards are new (so eg. 30 cars/day would always be 30).

Some amount of randomization was added to the card intervals, with a small amount at low boxes and a greater range at higher boxes. It's possible @JuTrip added those cards before I made this change to the SRS.

I think either way, some form of "throttling" I described, which has more effect on large intervals could solve the issue and give more control to users on the load of due cards, while still prioritizing recently learned cards for review.

It's a non trivial change as the SRS graph needs to account for it.

In the meantime the workaround is to chose a daily limit. The SRS will still sort the batch of due cards by last review time. So eg. review 10 cards, then select "END". The SRS page should confirm in green text "X reviews today", which helps if you want to review a fixed amount per day.
